lighthouse-core/gather/driver.js

```diff
@@ -115,15 +116,65 @@ class Driver {
   }
 
   /**
+   * Debounce enabling or disabling domains to prevent driver users from
+   * stomping on each other. Maintains an internal count of the times a domain
+   * has been enabled. Returns false if the command would have no effect (domain
+   * is already enabled or disabled), or if command would interfere with another
+   * user of that domain (e.g. two gatherers have enabled a domain, both need to
+   * disable it for it to be disabled). Returns true otherwise.
+   * @param {string} domain
+   * @param {string} action
+   * @return {boolean}
+   * @private
+   */
+  _checkDomainToggle(domain, action) {
```

Comment:
maybe something slightly more explicit for the name like `_shouldSendDomainCommand` or `_shouldToggleDomain`?